lib/pengine/tests/rules/pe_cron_range_satisfied_test.c

```diff
@@ -51,98 +51,130 @@ time_satisfies_year_spec(void **state) {
 static void
 time_after_year_spec(void **state) {
     run_one_test("2020-01-01",
-                 "<date_spec " PCMK_XA_ID "='spec' years='2019'/>",
+                 "<" PCMK_XE_DATE_SPEC " "
```

Comment:
Ken might not like this, I dunno. Depends on whether we consider want these tests to implicitly test the values of the string constants, or make sure the functions work correctly given the values of the string constants.